src/components/interface/PositionTable/components/PositionTableHead/PositionTableHead.tsx

```diff
@@ -110,7 +109,7 @@ const PositionTableHead: React.FunctionComponent<PositionTableHeadProps> = ({
             }}
           >
             <BulletLabel
-              sx={{ color: getFixedRateHealthTextColor(fixedRateHealthFactor) }}
+              sx={{ color: getHealthTextColor(fixedRateHealthFactor) }}
```

Comment:
it's important to note that there are two different types of health colours:
1. health colour of a position which is a function of their margin requirement (tells a user how safe their position is from being liquidated)
2. health colour (specific to liquidity providers), this one e.g. turns red when the current fixed rate in the amm is within the fixed rate range of the liquidity provider, yellow when it's close to a boundary of the range and green when it's within the range and not close to the boundaries (this one only applies to the positions in the lp positions screen)

Reply:
think it's important to make sure these are distinguishable in the code

Reply:
I kinda disagree to have same function with same logic just because it is used in different places. Think for example parseInt, you use it everywhere yet we don't call it parseIntForXFeature, parseIntForYFeature.

Perhaps we can think of better naming for getHealthTextColor that tries to unify both ideas in one?

Open for suggestions from your end!